pymeasure/instruments/instrument.py

```diff
@@ -22,16 +22,23 @@
 # THE SOFTWARE.
 #
 
+
+import abc
 import logging
+import threading
+from socket import error as socket_error
+from time import sleep, time
+
 import numpy as np
+
 from pymeasure.adapters.visa import VISAAdapter
 
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
 
 
 class DynamicProperty(property):
-    """ Class that allows managing python property behaviour in a "dynamic" fashion
```

Comment:
Lots of changes in `instrument.py` @brandonaltieri1. Could you call out (some of) the ones you made, so that code reviewers don't get distracted with formatting changes?

Reply:
No, thank you!